src/pages/claim-details/claim-details.tsx

```diff
@@ -0,0 +1,116 @@
+import { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+import { BaseLayout } from '../../components/layout';
+import BorderedBox, { Header } from '../../components/bordered-box';
+import ExternalLink from '../../components/external-link';
+import Timer, { DATE_FORMAT } from '../../components/timer';
+import ClaimActions from './claim-actions';
+import { useParams } from 'react-router';
+import { useChainData } from '../../chain-data';
+import { useUserClaimById } from '../../logic/claims';
+import { format } from 'date-fns';
+import { formatApi3, images } from '../../utils';
+import globalStyles from '../../styles/global-styles.module.scss';
+import styles from './claim-details.module.scss';
+
+interface Params {
+  claimId: string;
+}
+
+export default function ClaimDetails() {
+  const { claimId } = useParams<Params>();
+  const { data: claim, loading, loaded } = useUserClaimById(claimId);
+  const { provider } = useChainData();
+  if (!provider) {
+    return (
+      <BaseLayout subtitle={`Claim ${claimId}`}>
+        <p className={globalStyles.textCenter}>Please connect your wallet to see the claim details.</p>
+      </BaseLayout>
+    );
+  }
+
+  if (!claim) {
+    return (
+      <ClaimDetailsLayout claimId={claimId}>
+        <div className={styles.detailsHeader}>
+          <h4>Claim {claimId}</h4>
+        </div>
+        {loading && <p className={globalStyles.secondaryColor}>Loading...</p>}
+        {loaded && <p>Unable to find your claim with given id.</p>}
+      </ClaimDetailsLayout>
+    );
+  }
+
+  const evidenceHref = `https://ipfs.io/ipfs/${claim.evidence}`;
+  const transactionHref = claim.transactionHash ? `https://etherscan.io/tx/${claim.transactionHash}` : null;
```

Comment:
There is a helper fn `getEtherscanTransactionUrl` which already does this.

Reply:
Burak mentioned that showing the "Creation Transaction" (as per the design), doesn't make much sense, so this might not be a thing. I see the `getEtherscanTransactionUrl()` expects an Ethers `Transaction` type, so I reckon I will remove showing a transaction here until I'm sure we need to (and in that case, I will use that helper method 👍)